dist/origin-web-catalogs.js

```diff
@@ -7,13 +7,13 @@ webpackJsonp([ 0, 1 ], [ function(e, t) {
 }, function(e, t) {
-    e.exports = '<div class="center">\n  <div ng-if="!$ctrl.success && !$ctrl.error">\n    <div class="title">\n      Creating Application&hellip;\n    </div>\n  </div>\n  <div ng-if="$ctrl.success">\n    <div class="title">\n      Application Created <span class="fa fa-check success-check"></span>\n    </div>\n    <div class="sub-title center">\n      Continue to your project to view your application.\n    </div>\n    <div class="launch-service">\n      <!-- TODO: Correctly set baseProjectUrl -->\n      <a ng-href="{{$ctrl.selectedProject | projectUrl : $ctrl.baseProjectUrl}}" class="btn btn-primary order-btn">\n        View Project\n      </a>\n    </div>\n  </div>\n  <div ng-if="$ctrl.error">\n    <div class="title">Error <span class="fa fa-times text-danger"></span></div>\n    <div class="sub-title center">\n      <span ng-if="$ctrl.error.data.message">\n        {{$ctrl.error.data.message | upperFirst}}\n      </span>\n      <span ng-if="!$ctrl.error.data.message">\n        An error occurred creating the application.\n      </span>\n    </div>\n    <!-- TODO: Improve error message presentation -->\n    <ul ng-if="$ctrl.error.failure.length" class="failure-messages">\n      <li ng-repeat="failure in $ctrl.error.failure">\n        {{failure.data.message}}\n      </li>\n    </ul>\n  </div>\n  <div>\n    <a class="close-href" href="" ng-click="$ctrl.closePanel()">Close</a> this panel to browse other services.\n  </div>\n</div>\n';
+    e.exports = '<div class="center">\n  <div ng-if="!$ctrl.success && !$ctrl.error">\n    <div class="title">\n      Creating Application&hellip;\n    </div>\n  </div>\n  <div ng-if="$ctrl.success">\n    <div class="title">\n      Application Created <span class="fa fa-check success-check"></span>\n    </div>\n    <div class="sub-title center">\n      Continue to your project to view your application.\n    </div>\n    <div class="launch-service">\n      \x3c!-- TODO: Correctly set baseProjectUrl --\x3e\n      <a ng-href="{{$ctrl.selectedProject | projectUrl : $ctrl.baseProjectUrl}}" class="btn btn-primary order-btn">\n        View Project\n      </a>\n    </div>\n  </div>\n  <div ng-if="$ctrl.error">\n    <div class="title">Error <span class="fa fa-times text-danger"></span></div>\n    <div class="sub-title center">\n      <span ng-if="$ctrl.error.data.message">\n        {{$ctrl.error.data.message | upperFirst}}\n      </span>\n      <span ng-if="!$ctrl.error.data.message">\n        An error occurred creating the application.\n      </span>\n    </div>\n    \x3c!-- TODO: Improve error message presentation --\x3e\n    <ul ng-if="$ctrl.error.failure.length" class="failure-messages">\n      <li ng-repeat="failure in $ctrl.error.failure">\n        {{failure.data.message}}\n      </li>\n    </ul>\n  </div>\n  <div>\n    <a class="close-href" href="" ng-click="$ctrl.closePanel()">Close</a> this panel to browse other services.\n  </div>\n</div>\n';
```

Comment:
@jeff-phillips-18, I've got an issue when running `npm start build`.  &lt; and &gt; characters as part of an html comment are being escaped as hex codes, which results in invalid dist.  Have you seen this issue before?  I'm running the same node (v4.8.2) and npm (v2.15.11) versions as @spadgett, yet he can build from my source without the issue.